router.go

```diff
@@ -43,13 +48,13 @@ func assembleMiddleware() {
 }
 
 func assembleRoutes(build, hash string) {
-	buildstamp = buildstamp
+	buildstamp = build
 	githash = hash
 	var err error
 
 	// If bundling is enabled, read the RAML
 	// from bundle.go
-	if os.Getenv("BUNDLE_ASSETS") != "" {
+	if bundleAssets {
```

Comment:
Not sure about creating variables for values that are only used once. We could just do:

```go
if os.Getenv(""BUNDLE_ASSETS") == "1"
```
...as a call to an env variable that does not exist will just return an empty string.